docs/3.0.0-beta.x/community/contributing-guide.md

```diff
@@ -0,0 +1,75 @@
+# Open Development & Community Driven
+
+Strapi is an open-source project administered by [the Strapi team](https://strapi.io/company). We appreciate your interest and efforts in contributing to Strapi.
+
+There are many ways that you can contribute to Strapi. Contributions to Strapi and the greater Strapi Community can take the form of answering questions in one of the [many Strapi channels](/3.0.0-beta.x/community/contributing-guide.html#get-in-touch), writing articles, creating videos and otherwise publishing self-generated content.
+
+The two additional ways to contribute to Strapi include contributing **Documentation** and **Code**:
+
+- [Contribute in the form of Code Contributions](/3.0.0-beta.x/community/contribute-with-code.md)
+- [Contribute in the form of Documentation, Tutorials, or Articles](/3.0.0-beta.x/community/contribute-with-docs.md).
+
+Some people choose to support Strapi by [purchasing Swag from our Store](https://strapi.io/shop), (and then proudly wearing it!).
+
+Any way you choose to contribute to Strapi is welcomed and appreciated.
```

Comment:
All contributions are highly appreciated and welcomed.

Reply:
Yup. Changed. Thank you.